_topic_maps/_topic_map.yml

```
@@ -509,7 +509,7 @@ Topics:
     File: preparing-to-install-on-ibm-power
   - Name: Installing a cluster on IBM Power
     File: installing-ibm-power
-  - Name: Restricted network IBM Power installation
+  - Name: Disconnected environment IBM Power installation
```

This current change would make the terminology update while being minimally disruptive to the existing name structure. But if you're already changing the name of the page, perhaps you may as well align it with the other installer and IBM titles?
```suggestion
  - Name: Installing a cluster on IBM Power in a disconnected environment
```

Up to you. It would also better align with our [mod docs guidelines](https://redhat-documentation.github.io/modular-docs/#assembly-guidelines) that tell us to name assemblies with procedure modules using gerunds that describe the task.